frontend/tests/features/wheresMyGene/shareLink.test.ts

```diff
@@ -44,10 +44,13 @@ const SEXES = [
 ];
 const COMPARE = "disease";
 
+const CELL_TYPES = ["CD4-positive helper T cell"];
+
 const SHARE_LINK =
   `${TEST_URL}/gene-expression?` +
   // (thuang): Tissue params include a tissue id and a tissue name to test that we support both
-  "compare=disease&datasets=d8da613f-e681-4c69-b463-e94f5e66847f%2Cde2c780c-1747-40bd-9ccf-9588ec186cee&diseases=MONDO%3A0100096&ethnicities=unknown&sexes=PATO%3A0000383%2CPATO%3A0000384&tissues=blood%2CUBERON%3A0002048&genes=DPM1%2CTNMD%2CTSPAN6&ver=2";
+  // (seve): cellType params includes a incorrect cellType to test that we filter out invalid params
+  "compare=disease&datasets=d8da613f-e681-4c69-b463-e94f5e66847f%2Cde2c780c-1747-40bd-9ccf-9588ec186cee&diseases=MONDO%3A0100096&ethnicities=unknown&sexes=PATO%3A0000383%2CPATO%3A0000384&tissues=blood%2CUBERON%3A0002048&genes=DPM1%2CTNMD%2CTSPAN&cellTypes=DOG%2CCAT%2CCD4-positive+helper+T+cell&ver=2";
```

Comment:
can we create a constant for the incorrect cell type, so we can immediately identify that? Thank you!